packages/vite-plugin-cloudflare/src/workers-configs.ts

```diff
@@ -312,30 +315,46 @@ export function getWorkerConfig(
 
 	opts?.visitedConfigPaths?.add(configPath);
 
+	assert(
+		config.topLevelName,
+		`No top-level 'name' field provided in ${config.configPath}`,
+	);
+	assert(config.name, `No 'name' field provided in ${config.configPath}`);
```

Comment:
Perhaps something like this to help?

```suggestion
	assert(config.name, `No 'name' field provided in ${config.configPath}` + env !== undefined ? ` for ${env} environment.` : '.');
```

Reply:
Should I add this for all the config assertions? Could it be misinterpreted as meaning that a `name` has to be provided for the specific environment?

Reply:
I was thinking for all the assertions. Makes it clearer for the user.
These values are the inherited/overridden values, so if they don't exist at all then they were not at the top level either.
Actually I wonder if this particular one would ever be hit since we would fail on the topLevelName first.

Reply:
[fbe0fb3](https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/127/commits/fbe0fb3ecc9d76b557a459ec30ca5424c8b3d594)